var/spack/repos/builtin/packages/dialign-tx/package.py

```diff
@@ -19,19 +19,18 @@ class DialignTx(MakefilePackage):
 
     build_directory = "source"
 
-    conflicts("%gcc@6:")
+    # patches assembled from gentoo builds
+    # https://ftp.dimensiondata.com/mirror/gentoo-portage/sci-biology/dialign-tx/files/
+    patch("dialign-tx.patch")
```

Comment:
They're not, no, as I've merged the relevant parts of several different patch files. Differences in the handling of the `makefile` for spack prevent this being a straight liftover.

Reply:
@A-N-Other Can you extract this specific patch, add it as a PR to https://github.com/spack/patches (and ping me there), and then update this PR to point to that with a URL-based patch (once I merge it)?

I think that because `dialign-tx` is LGPL, the patches possibly are too. We'd like to avoid any potential issues by putting such patches in a separate Git repository.

The PR should place the patches in a new `dialign-tx/` directory in that repository.